randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -825,7 +828,17 @@ Extra - asset_id: collision_camera_012
               All of the following:
                   Gravity Suit
                   Space Jump or Simple IBJ
-              Spider Magnet and Use Spin Boost
+              All of the following:
+                  Spider Magnet
+                  Any of the following:
+                      Flash Shift or Use Spin Boost
```

Comment:
This and the other Flash addition you flash back into the same wall right? That would be merit for beginner wall jump. If you can flash directly to the other wall, then it's fine.